src/readme/rules.ts

```diff
@@ -360,12 +360,26 @@ const rules: IRule[] = [
   },
   {
     available: true,
+    eslintRule: 'no-unsafe-finally',
+    tslintRule: 'no-unsafe-finally',
+    category: 'Possible Errors',
+    description: 'disallow control flow statements in finally blocks (recommended)',
+    eslintUrl: 'http://eslint.org/docs/rules/no-unsafe-finally',
+    tslintUrl: 'https://palantir.github.io/tslint/rules/no-unsafe-finally',
+    provider: 'native',
+    usage: `~~~json
+    "use-isnan": true
+    ~~~`
+  },
+  {
+    available: true,
     eslintRule: 'use-isnan',
     tslintRule: 'use-isnan',
     category: 'Possible Errors',
     description: 'disallow comparisons with the value `NaN` (recommended)',
     eslintUrl: 'http://eslint.org/docs/rules/use-isnan',
-    provider: 'tslint-eslint-rules',
+    tslintUrl: 'https://palantir.github.io/tslint/rules/use-isnan',
+    provider: 'native',
```

Comment:
`src/docs/rules/useIsnanRule.md`is were we would put the link to the tslint rule.